src/server/game/Entities/Player/Player.cpp

```diff
@@ -13557,6 +13557,9 @@ uint32 Player::CalculateTalentsPoints() const
 
 bool Player::canFlyInZone(uint32 mapid, uint32 zone, SpellInfo const* bySpell) const
 {
+    if (!sScriptMgr->OnCanPlayerFlyInZone(this, mapid,zone,bySpell)) {
```

Comment:
bracket on a new line please.

Reply:
thanks for pointing that out. should be resolved now.